addons/website/static/src/scss/website.scss

```diff
@@ -1201,15 +1201,23 @@ header {
 }
 
 // Language selector
-.js_language_selector .dropdown-menu {
-    min-width: 0;
+.js_language_selector {
+    .dropdown-menu {
+        min-width: 0;
+    }
+    a.list-inline-item {
+        padding: 0.2em 0;
```

Comment:
@bvr-odoo Sorry but I still have a misalignement because of this. With a fixed **px** vertical padding there seems to be no problem, could you check please? Thank you

Reply:
> @bvr-odoo Sorry but I still have a misalignement because of this. With a fixed **px** vertical padding there seems to be no problem, could you check please? Thank you

Indeed, I can see a difference in Firefox, but in Chrome nothing change between 0.2em and 3px.